.github/workflows/ci.yaml

```diff
@@ -96,8 +96,6 @@ jobs:
           skip-pkg-cache: true
       - name: Build Kratos
         run: make install
-      - name: Run documentation tests
-        run: make test-docs
```

Comment:
https://github.com/ory/kratos/pull/3531/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L7

The npm command just executed `exit 0`.